packages/cli/src/commands/__tests__/daemon.test.ts

```
@@ -563,6 +563,8 @@ vi.mock("@fusion/core", async (importOriginal) => {
 });
 
 vi.mock("@fusion/dashboard", () => ({
+  // FNXC:TestInfrastructure 2026-07-13-10:25: Source files named-import these from @fusion/dashboard barrel; mock must surface them.
+  registerGithubTrackingHook: vi.fn(),
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)